packages/gatsby-theme-carbon/src/components/PageTabs/PageTabs.module.scss

```diff
@@ -1,7 +1,10 @@
 .tabs-container {
   margin: 0;
   width: 100%;
-  box-shadow: 0 -1px 0 rgba($ui-02, 0.25);
```

Comment:
Is this removed on purpose??

Reply:
Yeah it was emulating a border top which is now a proper border bottom in the page header.

Reply:
👍